tests/test_species.py

```diff
@@ -281,6 +281,17 @@ def test_contig_options(self):
             self.species.get_contig(
                 "chr22", exclusion_mask=[(0, 100)], length_multiplier=0.1
             )
```

Comment:
Shouldn't we also add a test here that if we `get_contig` with `use_species_gene_conversion` then the resulting rate and length agree with the species property (including if they are None)? That'll remove the need for the deepcopy in test_EscCol, I think?

Reply:
Ah, right - I got confused and thought this was testing all the contigs. I guess that could be a test up in `GenomeTestBase`? (or, is it `SpeciesTestBase`?)

Reply:
Hm - I added this test, and turned up an error (I think) - unless I'm wrong, I think that we're not actually disabling gene conversion by default. (we need additional logic in `species_contig( )`, I believe)

Reply:
Thanks for adding this. If I am not mistaken, the error appeared only because of a small bug, see my other comment.